WalletWasabi.Packager/Program.cs

```diff
@@ -588,12 +588,6 @@ private static void Publish()
 					{
 						Console.WriteLine("ERROR: Could not remove console from exe.");
 					}
-
-					// IF IT'S IN ONLYBINARIES MODE DON'T DO ANYTHING FANCY PACKAGING AFTER THIS!!!
-					if (OnlyBinaries)
-					{
-						continue; // In Windows build at this moment it does not matter though.
-					}
```

Comment:
Please don't touch this code. I think the comment there is clear enough to see this should not be removed.

Reply:
`if (OnlyBinaries)` is placed in the end of the `if (target.StartsWith("win"))` close so regardless of the state of `OnlyBinaries` the loop will continue. Maybe you should keep only the comment but you know better.